ZonPHP/install/par_edit.php

```diff
@@ -18,8 +18,8 @@
     $_POST['next_mail_threshold'] = $row['Waarde'];
 }
 
-
-$statoe = 1;
+//be careful with this one
+//$statoe = 1;
```

Comment:
For what is that needed anyway?
I just see assignments but no use of the variable